sites.json

```diff
@@ -898,6 +898,17 @@
     },
 
     {
+        "name": "CoinPayments",
+        "url": "https://www.coinpayments.net/help-support",
+        "difficulty": "impossible",
+        "notes": "",
```

Comment:
It's just CoinPayments policy, it says on the included URL "Account Deletions - CoinPayments accounts cannot be deleted, however you can discontinue using them at any time with no notice to us."